app/src/main/java/org/mozilla/fenix/utils/ClipboardHandler.kt

```diff
@@ -32,11 +33,12 @@ class ClipboardHandler(val context: Context) {
      */
     var text: String?
```

Comment:
Does it make sense to update the tests in ClipboardHandlerTest?

Reply:
Yeah, certainly, at the very least because they're failing :)